hasher-matcher-actioner/hmalib/common/actioner_models.py

```diff
@@ -48,6 +65,95 @@ class ActionRule:
 TUrl = t.Union[t.Text, bytes]
 
 
+@dataclass
+class ActionMessage(MatchMessage):
```

Comment:
Seems like a number of classes in this diff have the `to_aws_message` and `from_aws_message` functions. I think we should do a refactor and have these classes implement HMAConfig. That class handles the writing to and from dynamo and would help us remove a TON of boiler plate. Maybe a team discussion

Reply:
Seems like we can find a way to borrow some of the introspection / reflection code to where we have a helper to do this for any class. Definitely worth looking into at some point. V0 or V1, I'm not sure... But I'll add a TODO.